Makefile

```diff
@@ -347,10 +373,10 @@ metal-secrets:
 	-o yaml --dry-run | oc apply -n ocp -f -
 .PHONY: metal-secrets
 
-check:
+check-prow-config:
```

Comment:
We should nuke this or use `checkconfig` -- we do it in a presubmit so not sure we need it in `make`?

Reply:
Yeah, I don't think this even works in the current form (see the name of the job it is trying to find). It was not used from anywhere and could be nuked, but I did not want to make this PR any more complicated. I'll turn this to `checkconfig` once I start workign my way through the Makefile and migrate stuff.